src/herder/test/TransactionQueueTests.cpp

```diff
@@ -910,115 +805,6 @@ TEST_CASE_VERSIONS("TransactionQueue with PreconditionsV2",
     auto txSeqA1S3MinSeqLedgerGap = transactionWithV2Precondition(
         *app, account1, 3, 200, condMinSeqLedgerGap);
 
-    SECTION("gap valid due to minSeqNum")
```

Comment:
@sisuresh I've adjusted all tx queue/limiter tests to not rely on tx chains, except these few sections. I'm not sure there's much we can test here, given that these rely on several txs per account to be in the queue to do something meaningful. Could you please double check this and let me know if it makes sense?

Reply:
Yeah it makes sense to remove these, but we should still test the single transaction case (eg seqNum is 5 and someone submits a tx with minSeqNum 5 and seqNum 10). Are those cases still tested?

Reply:
Sounds good, I ended up updating "gap valid due to minSeqNum" section to check minSeqNum validity against the seqnums in the ledger. Let me know if this makes sense.